.github/workflows/test.yml

```diff
@@ -89,6 +89,23 @@ jobs:
     name: 'Test (${{ matrix.target }})'
     runs-on: ${{ matrix.platform }}
     steps:
+      - name: Install macos deps
+        if: ${{ startsWith(matrix.platform, 'macos') }}
+        run: |
+          brew install \
+            ninja
+
+      - name: Install linux deps
+        if: ${{ startsWith(matrix.platform, 'ubuntu') }}
+        run: |
+          sudo apt-get update
+          sudo apt-get install \
+            ninja-build
+
+      - name: Install windows deps
+        if: ${{ startsWith(matrix.platform, 'windows') }}
+        run: choco install -y ninja
+
```

Comment:
It looks like `ninja` should be added to the `coverage` workflow as well:
https://github.com/Open-CMSIS-Pack/cbuild/actions/runs/9566951112/job/26373623224

Reply:
[Install linux dependencies in coverage job](https://github.com/Open-CMSIS-Pack/cbuild/pull/243/commits/c25dcfd042cdc77936bc141cce4077ca24cf891d)